2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_092327-zguk9u0x/logs/debug.log
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_092327-zguk9u0x/logs/debug-internal.log
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:23:27,200 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-28 09:23:27,201 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: 0qtk51pf
2024-10-28 09:23:27,202 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/0qtk51pf
2024-10-28 09:23:27,202 INFO    MainThread:405 [jupyter.py:save_history():459] not saving jupyter history
2024-10-28 09:23:27,202 INFO    MainThread:405 [jupyter.py:save_ipynb():387] not saving jupyter notebook
2024-10-28 09:23:27,202 INFO    MainThread:405 [wandb_init.py:_jupyter_teardown():462] cleaning up jupyter logic
2024-10-28 09:23:27,202 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:23:27,202 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:23:27,202 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:23:30,411 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:23:30,411 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:23:30,416 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
2024-10-28 09:23:30,420 INFO    MainThread:405 [wandb_init.py:init():671] starting backend
2024-10-28 09:23:30,420 INFO    MainThread:405 [wandb_init.py:init():675] sending inform_init request
2024-10-28 09:23:30,421 INFO    MainThread:405 [backend.py:_multiprocessing_setup():104] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-10-28 09:23:30,422 INFO    MainThread:405 [wandb_init.py:init():688] backend started and connected
2024-10-28 09:23:30,428 INFO    MainThread:405 [wandb_run.py:_label_probe_notebook():1342] probe notebook
2024-10-28 09:23:30,437 INFO    MainThread:405 [wandb_init.py:init():783] updated telemetry
2024-10-28 09:23:30,437 INFO    MainThread:405 [wandb_init.py:init():816] communicating run to backend with 90.0 second timeout
2024-10-28 09:23:30,871 INFO    MainThread:405 [wandb_init.py:init():867] starting run threads in backend
2024-10-28 09:23:31,038 INFO    MainThread:405 [wandb_run.py:_console_start():2463] atexit reg
2024-10-28 09:23:31,038 INFO    MainThread:405 [wandb_run.py:_redirect():2311] redirect: wrap_raw
2024-10-28 09:23:31,038 INFO    MainThread:405 [wandb_run.py:_redirect():2376] Wrapping output streams.
2024-10-28 09:23:31,038 INFO    MainThread:405 [wandb_run.py:_redirect():2401] Redirects installed.
2024-10-28 09:23:31,039 INFO    MainThread:405 [wandb_init.py:init():911] run started, returning control to user process
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Current SDK version is 0.18.5
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Configure stats pid to 405
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/.config/wandb/settings
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from /home/jovyan/wandb/settings
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Loading settings from environment variables: {}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying setup settings: {'mode': None, '_disable_service': None}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_setup.py:_flush():79] Applying login settings: {'api_key': '***REDACTED***'}
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_log_setup():534] Logging user logs to /home/jovyan/wandb/run-20241028_094126-aw2wsuw7/logs/debug.log
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_log_setup():535] Logging internal logs to /home/jovyan/wandb/run-20241028_094126-aw2wsuw7/logs/debug-internal.log
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:_jupyter_setup():480] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f28ba204dc0>
2024-10-28 09:41:26,895 INFO    MainThread:405 [wandb_init.py:init():621] calling init triggers
2024-10-28 09:41:26,896 INFO    MainThread:405 [wandb_init.py:init():628] wandb.init called with sweep_config: {}
config: {}
2024-10-28 09:41:26,896 INFO    MainThread:405 [wandb_init.py:init():646] re-initializing run, found existing run on stack: zguk9u0x
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_finish():2158] finishing run gachon-university-fdslab/resnet18/zguk9u0x
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_atexit_cleanup():2426] got exitcode: 0
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_restore():2408] restore
2024-10-28 09:41:26,897 INFO    MainThread:405 [wandb_run.py:_restore():2414] restore done
2024-10-28 09:41:30,894 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():3975] rendering history
2024-10-28 09:41:30,894 INFO    MainThread:405 [wandb_run.py:_footer_history_summary_info():4007] rendering summary
2024-10-28 09:41:30,901 INFO    MainThread:405 [wandb_run.py:_footer_sync_info():3934] logging synced files
